tspng/test_extraction.py

```diff
@@ -2,14 +2,17 @@
 
 from tspng.extraction import extract_from_file,extract_from_files
 
-def test_hello_world():
-    print('Hello World!')
-
 def test_extract_from_file():
+    '''
```

Comment:
This is great, but docstrings on tests are not required. We will not convert the comments from tests into a help manual or hosted documentation.

Reply:
I will keep them in for now